index.js

```diff
@@ -1,13 +1,99 @@
+const $modal = document.querySelector('.modal');
+const $modalTitle = document.querySelector('#modal-title');
+const $pModal = document.querySelector('#p-modal');
+const $imgModal = document.querySelector('#img-modal');
+const $liModal = document.querySelectorAll('.li-modal');
+const $openModal = document.querySelectorAll('.button-project');
+const $closeModal = document.querySelector('.close-button');
+const frstImagePath = '/resources/work-section/masks/Snapshoot PortfolioTonic-mask.png';
+const scndImagePath = '/resources/work-section/masks/Snapshoot PortfolioMulti-Post-mask.png';
+const thrdImagePath = '/resources/work-section/masks/Snapshoot PortfolioTonic2-mask.png';
+const frthImagePath = '/resources/work-section/masks/Snapshoot PortfolioMulti-Post2-mask.png';
+let scrollEnabled = true;
+const projectsInfo = [{
+  name: 'Tonic',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: frstImagePath,
+}, {
+  name: 'Multi Post Stories',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: scndImagePath,
+}, {
+  name: 'Tonic',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: thrdImagePath,
+}, {
+  name: 'Multi-Post Stories',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: frthImagePath,
+}];
 
-let div = document.createElement('div');
-div.id = 'content';
-div.className = 'note';
+function selectProject(buttonIndex) {
+  const project = projectsInfo[buttonIndex];
+  $pModal.textContent = project.description;
+  $imgModal.setAttribute('src', project.image);
```

Comment:
- [ ] KIndly increase the spacing between the image element and the items beneath it tho match the design 💯.

- Your design
<img width="812" alt="image" src="https://github.com/SaveryIV/SaveryIV.github.io/assets/41163571/c9e7f34f-d2e7-42bc-be40-16cac457ad6a">

- Figma design
<img width="308" alt="image" src="https://github.com/SaveryIV/SaveryIV.github.io/assets/41163571/3c034b19-2baa-4b26-bb6e-c6caeef51754">